app/pages/project/instances/instance/tabs/NetworkingTab.tsx

```diff
@@ -209,7 +203,9 @@ export function NetworkingTab() {
           // we use primary to decide whether to show the IP in that row
           accessor="primary"
           id="external_ip"
-          cell={ExternalIpsFromInstanceName}
+          cell={({ value: primary }) =>
+            primary ? <ExternalIps {...instanceSelector} /> : <EmptyCell />
+          }
```

Comment:
This primary logic was inside the component before, but it's only relevant here, not in the table at the top. But you know... what if we just got rid of this column? The relation to the network interface is kind of tenuous to begin with. VPC name is different because each network interface can point to a different VPC. But external IPs really are instance level. So let's take them out of this table?

Reply:
I think it's an improvement! One other thing we could do is put a tooltip on IP to explain that it's an internal IP?

<img width="1160" alt="image" src="https://github.com/oxidecomputer/console/assets/3612203/bdcf90b6-5107-4a04-afbb-bf430daa03f2">